Cargo.toml

```diff
@@ -1,23 +1,25 @@
 [package]
-name = "rover"
```

Comment:
this lint is not my fave (name should be at the top!!)

Reply:
mmm y'know that is very fair. perhaps ill just get rid of it